common/components/WalletDecrypt/components/Web3.tsx

```diff
@@ -11,14 +11,14 @@ export const Web3Decrypt: React.SFC<Props> = ({ onUnlock }) => (
   <div className="Web3Decrypt">
     <div>
       <button className="Web3Decrypt-decrypt btn btn-primary btn-lg btn-block" onClick={onUnlock}>
-        {translate('ADD_MetaMask')}
+        {translateRaw('ADD_MetaMask')}
       </button>
     </div>
 
     <div>
       <NewTabLink
         className="Web3Decrypt-install btn btn-sm btn-default btn-block"
-        content={translate('Download MetaMask')}
+        content={translateRaw('Download MetaMask')}
```

Comment:
Should be a key, not a raw string. Might be a good idea to make this a variable'd translation so we could make it "Download $thing"